kit/media/src/UIImage/UIImage.native.tsx

```diff
@@ -26,25 +32,31 @@ export function prefetch(content: ImageURISource[] | ImageURISource): void {
     }
 }
 
-export const UIImage = React.forwardRef<Image, UIImageProps>(function UIImageForwarded(
-    { tintColor, ...rest }: UIImageProps,
+const UIImageSimple = React.forwardRef<RNImage, UIImageSimpleProps>(function UIImageSimple(
+    props: UIImageSimpleProps,
     ref,
 ) {
-    const theme = useTheme();
-    if (tintColor) {
+    if (props.tintColor) {
         /**
          * tintColor for some reason don't work properly with
          * react-native-fast-image, hence passing this prop
          * we force to use default <Image /> from RN
          */
-        return (
-            <Image
-                ref={ref}
-                {...rest}
-                style={[rest.style, tintColor != null ? { tintColor: theme[tintColor] } : null]}
-            />
-        );
+        return React.createElement(Image, { ...props, ref });
     }
     // @ts-expect-error
```

Comment:
Do you know why we need to silence the warning here?

Reply:
Not sure, but I guess because refs of RNImage and FastImage somehow incompatible :)